PyTado/interface.py

```diff
@@ -113,8 +113,8 @@ def _refresh_token(self):
             return False
 
         url = 'https://my.tado.com/oauth/token'
-        data = {'client_id' : 'tado-web-app',
-                'client_secret' : 'wZaRN7rpjn3FoNyF5IFuxg9uMzYJcvOoQ8QWiIqS3hfk6gLhVlG57j5YNoZL2Rtc',
+        data = {'client_id' : 'public-api-preview',
+                'client_secret' : '4HJGRffVR8xb3XdEUQpjgZ1VplJi6Xgw',
```

Comment:
It would be nice to get the values dynamically and not hardcoded.
At least the tado-web-app secret is available at https://my.tado.com/webapp/env.js.